packages/service-library/tests/test_archiving_utils.py

```diff
@@ -74,6 +78,32 @@ def get_dirs_and_subdris_in_path(path_to_scan: Path) -> List[Path]:
         yield temp_dir_path
 
 
+@pytest.fixture
+def file_content() -> str:
+    return "test" * 10
+
+
+@pytest.fixture
+def well_known_dir(tmp_path: Path, file_content: str) -> Path:
+    """Directory with well known structure"""
```

Comment:
I am not sure I like that name. what does it mean? a well known directory can be anything.
is there nothing more specific? what are you trying to mimic?